src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -76,16 +75,17 @@ export class AddCellAction extends Action {
 			}
 			if (context?.model) {
 				context.model.addCell(this.cellType, index);
+				context.model.sendNotebookTelemetryEvent(TelemetryKeys.NbTelemetryAction.AddCell, { cell_type: this.cellType });
 			}
 		} else {
 			//Add Cell after current selected cell.
 			const editor = this._notebookService.findNotebookEditor(context);
 			const index = editor.cells?.findIndex(cell => cell.active) ?? 0;
 			editor.addCell(this.cellType, index);
+			if (editor.model) {
```

Comment:
or just `editor.model?.sendNotebookTelemetryEvent`

Although why would the model not exist? We want to send the event each time we call AddCell so if this is a valid scenario then we may need to rethink where the helper function lives.

Reply:
I added the condition because a test was failing because of this. Let me check again, maybe we are not creating the model in the tests.

Reply:
Gotcha, yeah if that's the case then we should make a stub model instead preferably.